templates/index.html

```diff
@@ -24,7 +24,7 @@ <h1>Text and Image Extractor</h1>
             <button id="crop-button">Crop</button>
         </div>
     </div>
-    <script src="/static/form_handler.js"></script>
+    <script src="/app/templates/static/form_handler.js"></script>
```

Comment:
I'm seeing the following error in the browser consolde: `GET http://localhost:3000/app/templates/static/form_handler.js net::ERR_ABORTED 404 (Not Found)`

Reply:
Thank you for pointing that out. It looks like I made an error in the path. The correct path should still be under `/static` since FastAPI serves static files from that directory. I'll update the path to `/static/form_handler.js` in the `index.html` file.